### DISCUSSION TITLE
Save all settings for Taskbar, not just explorer settings

### DISCUSSION THREAD

**Discussion by highlanderc** [valinet/ExplorerPatcher#956] [Ideas]
Since the Release preview has huge fixes that I can't stand from the general release, we get a new build every week, sometimes two. Is there a way to save all settings? 
For instance. I use DU meter/Quick Launch and Battery bar pro toolbars. Can the system save all settings including layout too so once you re-import you are right back where you left off? 

Will to donate again for it.

Thanks

**Comment by valinet** [comment]
Yeah, I think if you want to export the entire taskbar configuration, you need to save the following things:

* DWORD: `HKEY_CURRENT_USER\Software\Classes\Local Settings\Software\Microsoft\Windows\CurrentVersion\TrayNotSIB` - Contains system tray icons settings
* DWORD: `HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\Streams\Desktop\TaskbarWinEP` - Contains toolbar layout information
* `HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\StuckRects3Legacy\Settings` - Contains main taskbar placement on screen and other things
* `HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\MMStuckRectsLegacy` - Contains secondary taskbar(s) placement on screen(s) and other things
